packages/hooking/manager.ts

```diff
@@ -181,3 +181,4 @@ export class HookManager {
 }
 
 export const hookManager = new HookManager();
+export const registerHook = hookManager.registerHook;
```

Comment:
As we found out previously, this does not work due to missing `this`. We could add a dedicated helper function here to simplify registering hooks. I pushed a commit showcasing this. Feel free to remove it afterwards.